tests/test_production_scenarios.py

```
@@ -0,0 +1,412 @@
+        # ストレージチェック
+        try:
+            self.storage._ensure_directories()
+            health_status["storage"] = True
+        except:
+            pass
+
+        # 設定チェック
+        try:
+            config = self.config_manager.get_default_config()
+            health_status["config"] = config is not None
+        except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)